libcaf_core/caf/detail/ring_buffer.test.cpp

```diff
@@ -0,0 +1,115 @@
+// This file is part of CAF, the C++ Actor Framework. See the file LICENSE in
+// the main distribution directory for license terms and copyright or visit
+// https://github.com/actor-framework/actor-framework/blob/master/LICENSE.
+
+#include "caf/detail/ring_buffer.hpp"
+
+#include "caf/test/caf_test_main.hpp"
+#include "caf/test/test.hpp"
+
+using namespace caf;
+
+using detail::ring_buffer;
+
+TEST("push_back adds element") {
+  ring_buffer<int> buf(3);
```

Comment:
Please take this to heart, I've mentioned it before: https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#es23-prefer-the--initializer-syntax

```suggestion
  ring_buffer<int> buf{3};
```